<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>show_profile</class>
 <widget class="QWidget" name="show_profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1568</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>1181</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qradialgradient(spread:repeat, cx:0.192, cy:0.801, radius:0.5, fx:0.41, fy:0.596, stop:0.0696517 rgba(190, 217, 225, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>311</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(153, 206, 211);
color: rgb(81, 108, 111);
font: italic 12pt &quot;Sitka Display&quot;;
gridline-color: rgb(0, 0, 0);
border-radius: 20px;

border-style: outset;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(49, 66, 68);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;username :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_user_pro">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>12</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>90</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(49, 66, 68);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Name :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_name_pro">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>82</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>160</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(49, 66, 68);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;bithday :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_age_pro">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>152</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>birthday</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>230</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(49, 66, 68);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;country :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_contry_pro">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>222</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>country</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>300</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(49, 66, 68);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Phone :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_phone_pro">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>292</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>phone number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>370</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(49, 66, 68);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Link :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_link_pro">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>362</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>link</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>321</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(153, 206, 211);
color: rgb(81, 108, 111);
font: italic 12pt &quot;Sitka Display&quot;;
gridline-color: rgb(0, 0, 0);
border-radius: 20px;
border-style: outset;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextBrowser" name="txtb_bio">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>241</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(203, 239, 240);
border-radius: 20px;
border-style: outset;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sitka Display'; font-size:12pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-style:normal;&quot;&gt;bio&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(49, 66, 68);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;bio :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_picture">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>221</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #ffffff ;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>70</y>
      <width>381</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(81, 108, 111);
background-color: rgb(153, 206, 211);
font: italic 12pt &quot;Sitka Display&quot;;
border-radius: 20px;
border-style: outset;
</string>
    </property>
    <property name="title">
     <string>Tweet :</string>
    </property>
    <widget class="QListWidget" name="list_tweet_show">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>351</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(203, 239, 240);
border-radius: 20px;
border-style: outset;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>250</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>following :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>follower:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_follower_show">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>300</y>
       <width>55</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_following_show">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>300</y>
       <width>55</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: ;
	color: rgb(99, 76, 80);
border: 2px solid rgb(157, 120, 127)  ;
border-radius: 20px;
border-style: outset;
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 rgb(237, 181, 191)
);
padding: 5px;
}

QPushButton:hover {
background:  qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #ffffff
);
}

</string>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
